OpenRA.Mods.Common/Widgets/Logic/MapChooserLogic.cs

```diff
@@ -43,7 +43,12 @@ public class MapChooserLogic : ChromeLogic
 			this.modData = modData;
 			this.onSelect = onSelect;
 
-			var approving = new Action(() => { Ui.CloseWindow(); onSelect(selectedUid); });
+			var approving = new Action(() =>
+			{
+				Ui.CloseWindow();
+				if (onSelect != null)
+					onSelect(selectedUid);
+			});
```

Comment:
Leave an empty line between this and `var canceling`

Reply:
Done